chatbot_attention_mechanism/chatbotpy.py

```diff
@@ -0,0 +1,21 @@
+import torch
+#from models.language_model import BigramLanguageModel
+from utils.data_loader import encode
```

Comment:
Can you please fix this line? The data_loader file does not contain any function named encode.